src/identity_matching/matcher.py

```diff
@@ -23,19 +23,25 @@ def match_emails(directory, seed):
     short_log = list()
     short_log_file = directory + "/shortlog.txt"
 
+    shortlog_timeout = False
     with open(short_log_file, "w+", encoding="latin-1", errors="replace") as outfile:
-        subprocess.call(["git", "-C", directory, "shortlog", "-se"], stdin=subprocess.DEVNULL, stdout=outfile, universal_newlines=True)
-
-    with open(short_log_file, "r", encoding="latin-1", errors="replace") as f:
-        for line in f.readlines():
-            try:
-                count, n, e = process_shortlog_line(line)
-                test_data = (count, n, re.sub(r"[<>]", "", e))
-                short_log.append(test_data)
-            except ValueError:
-                print("error ", line)
-
-    short_log = short_log[1:]
+        try:
+            subprocess.run(["git", "-C", directory, "shortlog", "-se"], stdout=outfile, universal_newlines=True, timeout=60)
+        except:
```

Comment:
I think, we should specifically catch subprocess.TimeoutExpired here.

Reply:
done